packages/vm/README.md

```diff
@@ -56,6 +56,14 @@ Supported `Istanbul` EIPs:
   SSTORE gas costs,
   PR [#590](https://github.com/ethereumjs/ethereumjs-vm/pull/590)
 
+#### EIP Support
+
+It is possible to individually activate EIP support in the VM. In order to do so, pass an array to the `eips` field of the VMs options, such as `new VM({eips: ['EIP2537']})`.
+
+Currently supported EIPs are:
+
+- [EIP-2537](https://eips.ethereum.org/EIPS/eip-2537): BLS precompiles
+
```

Comment:
Won't make this a blocker but please do not update `README` files along feature PRs yet. Let's please keep this to the release PRs once the releases are actually done. Then one can go through the - now already existing 😄  - release notes and do a consistent update on the `README` files. This will otherwise confuse users of the latest releases published on npm.

Reply:
We do update the changelog right? So the changelog does have new versions (which are not available on npm yet, which is also a bit confusing), but we keep the readme at the latest published version?